.gitignore

```diff
@@ -1,3 +1,11 @@
+.yarn/*
+!.yarn/patches
+!.yarn/plugins
+!.yarn/releases
```

Comment:
Can we do this without vendoring dependencies?

Reply:
I think so. This is added by default for yarn 2+, but should work fine if we don't commit these files. 

Will update the PR soon.

Reply:
The docs for newer versions of yarn (that I'm using here for workspaces) say to put this in `.gitignore`:

```
.pnp.*
.yarn/*
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```

This is following instructions for opting out of the new "zero installs" feature, and instead having a regular node_modules folder. https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Looks like [react-spring](https://github.com/pmndrs/react-spring) also commits these files in `.yarn`.

I'm happy to do whatever. We can try ignoring these files anyway - LMK

Reply:
I'd like to avoid vendoring files. I don't see any benefit to including a bunch of yarn files in this project.